common/lib/xmodule/xmodule/progress.py

```diff
@@ -116,12 +116,15 @@ def __ne__(self, other):
         return not self.__eq__(other)
 
     def __str__(self):
-        ''' Return a string representation of this string.
+        '''Return a string representation of this string. Rounds results to
```

Comment:
Huh. Can you make one of the test cases exhibit the rounding behavior so it is more obvious?